src/main/java/org/candlepin/hostedtest/HostedTestResource.java

```diff
@@ -305,8 +316,10 @@ public SubscriptionInfo createSubscription(
     @GET
     @Produces(MediaType.APPLICATION_JSON)
     @Path("/subscriptions")
-    public Collection<? extends SubscriptionInfo> listSubscriptions() {
-        return this.datastore.listSubscriptions();
+    public Collection<SubscriptionDTO> listSubscriptions() {
```

Comment:
Couple things here:
- when translating a collection like this, we should use the streamMapper, so we don't have to do the class resolution thing per-instance
- We could change the output to a Stream\<SubscriptionDTO\> here and save the collection step

Reply:
Done.